lib/ts/recipe/emailpassword/index.ts

```diff
@@ -112,6 +116,7 @@ export default class Wrapper {
     ): Promise<
         | { status: "OK"; user: User; recipeUserId: RecipeUserId }
         | { status: "WRONG_CREDENTIALS_ERROR" }
+        | { status: "USER_BANNED_ERROR" }
```

Comment:
probably also want to output the user object alongside this. Cause this way, devs can know which user id this belongs too